cookiecutter.json

```diff
@@ -1,11 +1,9 @@
 {
     "full_name": "Your name",
-    "email": "Your address email (eq. you@example.com)",
-    "github_username": "Your github username",
+    "email": "Your email address (eq. you@example.com)",
```

Comment:
by the way: is `eq` a standard abbreviation for `e.g.`?

Reply:
hm…I never noticed it says `eq.` here. I've no idea what that means :)